src/main/webapp/matchdashboard.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;1,400&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <!-- ========================================================
+        Menu
+         ======================================================== -->
+    <div class="menu">
+      <div class="profile-bar">
+        <img class="img-small" src="./assets/svgs/account.svg" alt="" />
+        <a href="" id="user-profile">Username</a>
```

Comment:
You still have duplicate `user-profile` IDs in this file and in some of the others. That's one of the downsides of having duplicate HTML code in multiple files, you have to remember to update all of them. Maybe a follow-up PR can refactor this common `menu` div so contributors don't have to remember to update all of them when making menu changes.